### PR TITLE
Task-3

### PR DESCRIPTION
**Pull request by serg-ksv**

### CODE REVIEW

File: src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        reviewDto1.setHelpfulnessDenominator(1);
+        reviewDto1.setScore(5);
+        var time1 = Instant.ofEpochMilli(1303862400)
+                .atZone(ZoneId.systemDefault()).toLocalDateTime();
+        reviewDto1.setTime(time1);
+        reviewDto1.setSummary("Good Quality Dog Food");
+        reviewDto1.setText("I have bought several of the Vitality canned dog food "
+                + "products and have found them all to be of good quality. The product "
+                + "looks more like a stew than a processed meat and it smells better. "
+                + "My Labrador is finicky and she appreciates this product better than  most.");
```
Comment (dmytro-hryhoruk) [inline src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java:40]: we don't need this whole text for testing. Let's cut this field to one small sentence

File: src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        reviewDto2.setHelpfulnessNumerator(0);
+        reviewDto2.setHelpfulnessDenominator(0);
+        reviewDto2.setScore(1);
+        var time2 = Instant.ofEpochMilli(1346976000)
+                .atZone(ZoneId.systemDefault()).toLocalDateTime();
+        reviewDto2.setTime(time2);
+        reviewDto2.setSummary("Not as Advertised");
+        reviewDto2.setText("Product arrived labeled as Jumbo Salted Peanuts...the peanuts "
+                + "were actually small sized unsalted. Not sure if this was an error or if "
+                + "the vendor intended to represent the product as \"Jumbo\".");
```
Comment (dmytro-hryhoruk) [inline src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java:55]: same

File: src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java (hunk 1 above)
Comment (serg-ksv) [reply src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java]: done

File: src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java (hunk 2 above)
Comment (serg-ksv) [reply src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java]: done

**Pull request merged**